skate/source/includes/_intro.md

```diff
@@ -0,0 +1,111 @@
+    {
+      "id": 5,
+      "user_id": 1,
+      "score": 220,
+      "tag": "level two",
+      "created_at": "2015-07-12T12:16:22.018"
+    }
+  ],
+  "meta": {
+    "total": 55,
+    "links": {
+      "prev": "https://tmwild.com/api/leaderboards?offset=1&page_size=1",
+      "next": "https://tmwild.com/api/leaderboards?offset=35&page_size=1"
+    }
+  }
+}
+```
+
+Each response will be JSON formatted. The object will contain `data` and `meta`.
+Data will contain the data requested, or information about the action performed.
+Meta is information about the response itself. See the meta section for more
+info. TODO link. All example responses in the rest of this document will show
+the `data` object and not the full response object.
+
+##Meta
+
+> Example meta object:
+
+```json
+{
+  "total": 55,
+  "links": {
+    "prev": "https://tmwild.com/api/leaderboards?offset=5&page_size=5",
+    "next": "https://tmwild.com/api/leaderboards?offset=15&page_size=5"
+  }
+}
+```
+
+Each response will include a `meta` object, which contains info about the response
+itself. For requests returning lists the `meta` object will contain a `total`
+key which is set to the total number of objects the request describes. If this
+number is large, it will be different than the number of objects returned, as
+the results are paginated. In this case, the `meta` object will also contain
+a `links` object. This object will contain `next` and/or `previous`, which are
+links that point to the next and previous page of responses, respectively. If
+there are no more results or if it is the first response the `next` or `previous`
+keys will not exist, respectively.
+
+##Pagination
+With requests that ask for lists in response, there are two url parameters you
+can pass: `page_size`, which indicates how many elements should be returned per
+page, and `offset`, which indicates which element to start at. The default
+`page_size` is 5, and the maximum is 25. The default `offset` is `1`, which
+refers to the first element.
+
+<aside class="warning">
+Note that
+<code class="prettyprint">page_size</code>
+ is not necessarily the number of objects returned. For
+example, if there are 10 total objects, and you request with
+<code class="prettyprint">page_size</code> =
+<code class="prettyprint">5</code>
+and
+<code class="prettyprint">offset</code> =
+<code class="prettyprint">8</code>
+, only three elements will be in the response.
+</aside>
+
+These parameters will not be shown in the example requests for brevity.
+
+
+##Rate Limit
+The number of requests that can be made through the API is currently limited to
+(TODO) per (TODO) per OAuth token. If you exceed the rate limit, you will get
+a `429` error response. For all the errors that can be returned, see the error
+section below. TODO link.
+
+
+##Date Format
+All of the dates in requests and responses are to be interpretted as UTC. They
+must be in the ISO format `YYYY-MM-DDTHH:mm:ss` for example: `"1994-03-06T23:45:11"`.
```

Comment:
I said UTC, not unix time. My change was to remove the timezone part, requiring everything to be UTC. The format can stay ISO, just sans timezone.